internal/testdata/bitcoin/transaction.go

```diff
@@ -220,6 +220,34 @@ var Transactions = map[string]struct {
 	},
 }
 
+// TxMerkleProof holds details of the transaction Merkle proof data used as a
```

Comment:
Can we add a link to the transaction with this Merkle proof?

https://blockstream.info/testnet/api/tx/72e7fd57c2adb1ed2305c4247486ff79aec363296f02ec65be141904f80d214e